docs/api/dashboard.md

```diff
@@ -306,41 +322,9 @@ Update a Dashboard, and optionally the `dashcards` and `tabs` of a Dashboard. Th
 
 ### PARAMS:
 
-*  **`parameters`** nullable sequence of parameter must be a map with :id and :type keys
-
-*  **`points_of_interest`** nullable string
-
-*  **`description`** nullable string
-
-*  **`archived`** nullable boolean
-
-*  **`dashcards`** nullable value must be seq of maps in which ids are unique
-
-*  **`collection_position`** nullable value must be an integer greater than zero.
-
-*  **`tabs`** nullable value must be seq of maps in which ids are unique
-
-*  **`show_in_getting_started`** nullable boolean
-
-*  **`enable_embedding`** nullable boolean
-
-*  **`collection_id`** nullable value must be an integer greater than zero.
-
-*  **`dash-updates`** 
-
-*  **`name`** nullable value must be a non-blank string.
-
-*  **`width`** nullable enum of fixed, full
-
-*  **`caveats`** nullable string
-
-*  **`embedding_params`** nullable value must be a valid embedding params map.
-
-*  **`cache_ttl`** nullable value must be an integer greater than zero.
-
-*  **`id`** value must be an integer greater than zero.
+-  **`id`** value must be an integer greater than zero.
 
-*  **`position`** nullable value must be an integer greater than zero.
+-  **`dash-updates`** map where {:name (optional) -> <nullable value must be a non-blank string.>, :description (optional) -> <nullable string>, :caveats (optional) -> <nullable string>, :points_of_interest (optional) -> <nullable string>, :show_in_getting_started (optional) -> <nullable boolean>, :enable_embedding (optional) -> <nullable boolean>, :embedding_params (optional) -> <nullable value must be a valid embedding params map.>, :parameters (optional) -> <nullable sequence of parameter must be a map with :id and :type keys>, :position (optional) -> <nullable value must be an integer greater than zero.>, :width (optional) -> <enum of fixed, full>, :archived (optional) -> <nullable boolean>, :collection_id (optional) -> <nullable value must be an integer greater than zero.>, :collection_position (optional) -> <nullable value must be an integer greater than zero.>, :cache_ttl (optional) -> <nullable value must be an integer greater than zero.>, :dashcards (optional) -> <nullable value must be seq of maps in which ids are unique>, :tabs (optional) -> <nullable value must be seq of maps in which ids are unique>}.
```

Comment:
format seems messed up

Reply:
We're relying on malli's `describe` function to produce the string here. We should tackle this in a subsequent PR.